decidim-admin/app/commands/decidim/admin/create_import.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class CreateImport < Rectify::Command
+      def initialize(form)
+        @form = form
+      end
+
+      def call
+        return broadcast(:invalid) if form.invalid?
+        return broadcast(:invalid) unless form.creator
```

Comment:
Could this be moved to a form validation?

Reply:
Yep. I removed creator check from the command.